In [21]:
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
import warnings
from math import factorial


## Задание 1 Случайная непрерывная величина A имеет равномерное распределение на промежутке (200, 800]. Найдите ее среднее значение и дисперсию.

In [1]:
M = (200+800)/2
print('Среднее значение', M)

Среднее значение 500.0


In [3]:
D = ((800-200)**2)/12
print('Дисперсия ', D)

Дисперсия  30000.0


## Задание 2 О случайной непрерывной равномерно распределенной величине B известно, что ее дисперсия равна 0.2.Можно ли найти правую границу величины B и ее среднее значение зная, что левая граница равна 0.5? Если да, найдите ее.

D = ((b-a)**2)/12
12D = b**2 -2*b*a + a**2
2.4 = b**2-2b*a +0.25
2.15=b**2-2*b*a
b**2-2*b*a - 2.15 =0
b**2-2*b*0.5 - 2.15 =0
b**2-b - 2.15 =0
Diskr = 1**2 +4*1*2.15
Diskr = 9.6
x1,2 = (-b +- sqrt(Diskr) )/2a
x1 = (1 + (1+4*1*2.15)**0.5)/2
x2 = (1 - (1+4*1*2.15)**0.5)/2


In [14]:
x1 = (1 + (1+4*1*2.15)**0.5)/2
x2 = (1 - (1+4*1*2.15)**0.5)/2
print(x1,x2)

2.049193338482967 -1.0491933384829668


-1.0491933384829668 не подходит, поскольку мы знаем что левая гравница это 0.5


In [15]:
M = (2.049193338482967 + 0.5)/2
print('Ответ, правая граница величины B -', x1, 'Среднее значение - ', M)

Ответ, правая граница величины B - 2.049193338482967 Среднее значение -  1.2745966692414834


## Задание 3 Непрерывная случайная величина X распределена нормально и задана плотностью распределения
f(x) = (1 / (4 * sqrt(2*pi))) * (exp(-((x+2)**2) / 32))
Найдите:
а). M(X)
б). D(X)
в). std(X) (среднее квадратичное отклонение)

так как мы знаем что Х распределена нормально -  M(x) = 2, sigma = 4, D(X) = 16

## Задание 4 Рост взрослого населения города X имеет нормальное распределение.
Причем, средний рост равен 174 см, а среднее квадратичное отклонение равно 8 см.
Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:
а). больше 182 см
б). больше 190 см
в). от 166 см до 190 см
г). от 166 см до 182 см
д). от 158 см до 190 см
е). не выше 150 см или не ниже 190 см
ё). не выше 150 см или не ниже 198 см
ж). ниже 166 см.


а) P(X>182)= Ф(беск) - Ф(182)

Ф(беск) - Ф(182) = 1/2 - (Ф((182-174)/8) = 1/2 - Ф(1) = 1/2 - 0,3413

In [21]:
1/2 - 0.3413

0.1587

б)P(X>190)=Ф(беск) - Ф(190)

Ф(беск) - Ф(190) = 1/2 - (Ф((190-174)/8) = 1/2 - Ф(2) = 1/2 - 0,4772

In [23]:
1/2 - 0.4772

0.022799999999999987

в)P(166<X<190)= Ф(190) - Ф(166) = Ф(190-174/8) - Ф(166-174/8) = Ф(2)+Ф(1)

In [25]:
0.4772 + 0.3413

0.8185

г) P(166<X<182)= Ф(1)+Ф(1)

In [26]:
0.3413+0.3413

0.6826

д) P(158<X<190)= Ф(190) - Ф(158) = Ф(2)+Ф(2)

In [27]:
0.4772+0.4772

0.9544

e)  P(X<150)=F(150) = Ф(150) - Ф(-беск) = -Ф(3) + 1/2

In [28]:
-0.49865 + 1/2

0.0013500000000000179

P(X>190)=0.022799999999999987 из б) задания

In [8]:
print('не выше 150 см или не ниже 190 см',0.0013500000000000179+0.02279999999999993)

не выше 150 см или не ниже 190 см 0.02414999999999995


ё) P(X<150) = 0.0013500000000000179; P(X>198) = Ф(беск) - Ф(198)= 1/2 - Ф(3) эти цифры должны совпадать как так значения за 3сигма

In [29]:
1/2 -0.49865

0.0013500000000000179

In [30]:
0.0013500000000000179 + 0.0013500000000000179

0.0027000000000000357

ж) P(X<166)=F(166) =Ф(166) - Ф(-беск) = -Ф(1) + 1/2    (Аналогично заданию а)

In [32]:
-0.3413+1/2

0.1587

## Задание 5 На сколько сигм (средних квадратичных отклонений) отклоняется рост человека, равный 190 см, от математического ожидания роста в популяции, в которой M(X) = 178 см и D(X) = 25 кв.см?

In [44]:
(190-178)/25**0.5

2.4

Ответ на 2.4 сигмы